### PR TITLE
Add configuration to the readme

### PR DESCRIPTION
**Pull request by carlossg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,51 @@
...
+                "s3:DeleteObject",
+                "s3:ListObjects"
+            ],
+            "Resource": "arn:aws:s3:::my-bucket-name/some/path/*"
+        }
+    ]
+}
+```
+
+Then, run the Jenkins master with the environment variables. Note the `/` at the end of `S3_DIR`
```
Comment (v1v) [inline README.md]: Good point! It took me a while until I realise i was missing it 👍

**Comment by v1v** [comment]
🐝  I can confirm that particular policy works like a charm

File: README.md (hunk 1 above)
Comment (jglick) [reply README.md]: We should report an error if it is omitted. (though I guess the empty string could be a valid “dir” as well?)

File: README.md [hunk 2]
```diff
@@ -3,6 +3,51 @@
...
+
+```
```
Comment (jglick) [inline README.md]: maybe to emphasize:

`````markdown
```bash
export VAR=val
```
`````

Comment (carlossg) [reply README.md]: This is likely to change, will do a follow-up

**Pull request merged**